src/cronMatcher.ts

```diff
@@ -170,3 +174,14 @@ export function parseCrontabRanges(matches: string[], source: string) {
   );
   return { minutes, hours, dates, months, dows };
 }
+
+export const parseCronRangeString = (
+  pattern: string,
+  source: string = "",
```

Comment:
We should only add `in ${source}` if there is a source; so either remove the `= ""` here or handle the `source` missing in the error message and make sure `parseCrontabRanges` is also happy with no source.

```suggestion
  source: string,
```